Atomic/util.py

```diff
@@ -842,21 +842,31 @@ def wrapper(*args, **kwargs):
 
 
 class InstallData(object):
-    if not os.path.exists(ATOMIC_INSTALL_JSON):
-        open(ATOMIC_INSTALL_JSON, 'a').close()
-
-    install_file_handle = open(ATOMIC_INSTALL_JSON, 'r')
 
     @staticmethod
-    def _read_install_data(file_handle):
+    def get_install_file_handle():
```

Comment:
Does this need to be a function? Seems like we can just merge it into `_read_install_data()` so that you can just use context managers there to open and close, e.g.:

```python
try:
  with open(ATOMIC_INSTALL_JSON, 'r') as f:
    return json.load(f)
except IOError as e:
  if e.errno == errno.ENOENT:
    return {}
  raise e
```

?